DependencyInjection/Configuration.php

```diff
@@ -42,6 +42,12 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('token_ttl')
                     ->defaultValue(3600)
+                    ->validate()
+                        ->ifTrue(function ($ttl) {
+                            return !is_numeric($ttl);
```

Comment:
we should allow `null` values here as per https://github.com/lexik/LexikJWTAuthenticationBundle/pull/117